src/wp-admin/users.php

```diff
@@ -742,17 +747,18 @@
 		endif;
 		?>
 
-		<?php if ( isset( $errors ) && is_wp_error( $errors ) ) : ?>
-			<div class="error">
-				<ul>
-				<?php
-				foreach ( $errors->get_error_messages() as $err ) {
-					echo "<li>$err</li>\n";
-				}
-				?>
-				</ul>
-			</div>
-			<?php
+		<?php
+		if ( isset( $errors ) && is_wp_error( $errors ) ) :
+			$error_message = '';
+			foreach ( $errors->get_error_messages() as $err ) {
+				$error_message .= "<li>$err</li>\n";
+			}
+			wp_admin_notice(
+				$error_message,
```

Comment:
```suggestion
				'<ul>' . $error_message . '</ul>',
```
Need to wrap this one with `<ul>` tags.